src/input.rs

```diff
@@ -199,6 +201,7 @@ pub struct Config {
     ulimit: Option<rlimit::rlim>,
     scan_order: Option<ScanOrder>,
     command: Option<Vec<String>>,
+    scripts: Option<String>,
```

Comment:
Since we know what to expect for these scripts we could use an `enum` here with the three options we expect. What do you think?

Reply:
This has the advantage that Rust will be able to enforce that we are always taking into account he three possible variants in our code.

Reply:
All right, we can build an enum around the 3 values, now I just handle it like [ &_ => Ok(scripts_to_run),](https://github.com/RustScan/RustScan/pull/242/files#diff-5062b06fea1a548c03f86fb29d0fa7d9R103) :)

Reply:
Also, the commandline parser allows only the 3 options 
`#[structopt(long, possible_values = &["none", "default", "custom"], case_insensitive = true, default_value = "default")]`